### PR TITLE
RHDEVDOCS-3365 - Add cluster logging supported versions table.

### PR DESCRIPTION
**Pull request by libander**
- Aligned team: Dev Tools
- For branches: enterprise-4.7, 4.8
- https://issues.redhat.com/browse/RHDEVDOCS-3365
- Direct link to doc preview: : https://deploy-preview-38824--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes
- SME review: @alanconway 
- QE review: @anpingli 
- Peer review: @Srivaralakshmi 
- Please merge

### CODE REVIEW

File: modules/cluster-logging-supported-versions.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.{product-title} version support for Red Hat OpenShift Logging (RHOL)
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-supported-versions.adoc:1]: Suggestion:

- I do not see any module file metadata here. Please refer to this section: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata and if you plan to retain the module, implement the guidelines referenced here. 

But if you decide to move the content from the cluster-logging-supported-versions.adoc module and put it under the Supported versions heading, then there is no action required as you will be deleting the include statement Line 19 and the cluster-logging-supported-versions.adoc file from the modules folder.
- Also, if you decide that Line 13 (Red Hat OpenShift Logging 5.1) and Line 33 (Red Hat OpenShift Logging 5.0) can go as modules, then ensure to implement the guidelines referenced here.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@libander I have left some suggestions, PTAL. Thanks!

**Comment by Srivaralakshmi** [comment]
> * Direct link to doc preview: https://deploy-preview-38824--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-1-0

Please ensure to add the direct doc preview link in the PR description. I have added it to this one ^^.

File: logging/cluster-logging-release-notes.adoc [hunk 2]
```diff
@@ -14,9 +14,10 @@ Red Hat is committed to replacing problematic language in our code, documentatio
```
[the remark this answers is not in the document]
Comment (libander) [reply logging/cluster-logging-release-notes.adoc]: Thank you!

**Comment by Srivaralakshmi** [comment]
> * Direct link to doc preview: [Supported Versions](https://deploy-preview-38824--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes#supported-versions)

@libander Again, that same comment: Please ensure to add the direct doc preview link in the PR description. I have added it to this one ^^.

**Review by Srivaralakshmi — APPROVED** [review]
@libander Thanks for addressing the comments. Approved it erroneously. Then checked the missing anchor ID and have left a suggestion, PTAL.

File: logging/cluster-logging-release-notes.adoc [hunk 3]
```diff
@@ -10,47 +10,20 @@ toc::[]
...
-OpenShift Logging 5.1 runs on {product-title} 4.7 and 4.8.
-
-.Advisories
-
-The following advisories are available for OpenShift Logging 5.1.x:
-
-* link:https://access.redhat.com/errata/RHBA-2021:3545[RHBA-2021:3545 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.2]
-* link:https://access.redhat.com/errata/RHBA-2021:2885[RHBA-2021:2885 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.1]
-* link:https://access.redhat.com/errata/RHBA-2021:2112[RHBA-2021:2112 - Bug Fix Advisory. OpenShift Logging Bug Fix Release 5.1.0]
+== Supported Versions
```
Comment (Srivaralakshmi) [inline logging/cluster-logging-release-notes.adoc:13]: @libander Needs an anchor ID for this section heading. Please refer to: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]

**Pull request merged**